File: DSA/Satyam_Jhinkwan_590012742/Day6/2.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    int n;
    cout << "Enter size of array: ";
    cin >> n;
    vector<int> arr(n);
    cout << "Enter the elements:\n";
    for (auto &x : arr)
        cin >> x;
    for (int i = 0; i < n - 1; i++)
    {
        if (arr[i] >= arr[i + 1])
        {
            cout << "False";
            return 0;
        }
    }
    cout << "True";
    return 0;
}

File: DSA/Sandeep_Jha_590014310/week3/day16/2.py
def count_occur(arr,target):
    count = 0
    for e in arr:
        if e == target:
            count += 1
    return count

print(count_occur(arr=list(map(int,input("enter elements space-seperated: ").replace(',', ' ').split())),target=int(input("enter target: "))))